KnockKnock-iOS/Extension-Utilities/UIKit/UILabel+Ext.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  UILabel+Ext.swift
+//  KnockKnock-iOS
+//
+//  Created by Daye on 2022/06/23.
+//
+
+import UIKit
+
+extension UILabel {
+
+  var visibleTextLength: Int {
```

Comment:
```suggestion
  /// Summary
  var visibleTextLength: Int {
```
`///`   주석으로 Summary 를 적어주시면 더 무슨일을 하는지 알기 쉬워 보입니다! 

그렇게 되면 해당 함수나 프로퍼티에 option + 클릭을 하게되면 Summary가 나오기 때문에 좋습니다. 
구두로 나중에 또 알려드릴게요